Functions/xASL_stat_PrintStats.m

```diff
@@ -142,13 +117,13 @@
                 warning(['Missing data, skipping printing data for ' x.SUBJECTS{iSubject} '_' x.SESSIONS{iSession}]);
             else
                 % print subject name
-                fprintf(x.S.FID,'%s\t', x.S.SubjectSessionID{iSubjectSession, 1});
+                x.modules.population.(thisFile){iSubjectSession+2,1} = x.S.SubjectSessionID{iSubjectSession, 1}; % I'm not 100% sure this is correct, how do I test this?
```

Comment:
@jan-petr: I haven't properly tested this part, yet. Other than that it seems to work just fine.